crystal-map-processor/src/main/java/com/schwarz/crystalprocessor/model/accessor/CblGenerateAccessorHolder.kt

```diff
@@ -14,24 +14,26 @@ class CblGenerateAccessorHolder(
 ) {
 
     fun accessorFunSpec(): FunSpec? {
-
         if (memberFunction != null) {
             val methodBuilder = FunSpec.builder(memberFunction.name).addAnnotation(JvmStatic::class)
 
             if (memberFunction.isSuspend) {
                 methodBuilder.addModifiers(KModifier.SUSPEND)
             }
             val callParams = arrayListOf<String>()
+
             memberFunction.parameters.forEach {
                 callParams.add(it.name)
                 methodBuilder.addParameter(it.name, it.type)
             }
 
-            methodBuilder.addStatement(
-                "return %T.%N(${callParams.joinToString()})" + System.lineSeparator(),
-                sourceClassTypeName,
-                memberFunction.name
-            )
+            methodBuilder
+                .returns(memberFunction.returnTypeName)
```

Comment:
always expected "Unit" as the return type, should now specify the proper return type